### DISCUSSION TITLE
Deepgram API issue, critical.

### DISCUSSION THREAD

**Discussion by cowanAI** [deepgram/community#607] [General help] [answered]
### Which Deepgram product are you using?

None

### Details

<img width="846" alt="image" src="https://github.com/deepgram/community/assets/128092052/b4d24c64-8fbb-49ca-be76-710a8e0b14df">

receiving "none" 

INFO:root:File format determined: webm
None
None
None
ERROR:root:No audio detected in the recording: No audio was detected. Please verify your microphone is working as expected on your browser settings.
Traceback (most recent call last):
  File "/Users/ferncowan/DeepCura-Enterprise/backend/app/routers/transcripts.py", line 165, in transcribe_audio
    response_paragraphs = transcript_response["results"]["channels"][0]["alternatives"][0].get("paragraphs", [])
                          ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

_No response_

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

_No response_

### If possible, please attach your code or paste it into the text box.

_No response_

### If possible, please attach an example audio file to reproduce the issue.

_No response_

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
Please take a look at the latest release posted today. I believe the issue with accessing elements/properties as a dictionary have been resolved.
https://github.com/deepgram/deepgram-python-sdk/releases/tag/v3.1.5
